warehouse/templates/pages/help.html

```diff
@@ -95,6 +95,14 @@ <h3 id="project-name-claim" class="common-question">How do I claim an abandoned
       <p>There is currently no established process for performing this administrative task that is explicit and fair for all parties. However, one is currently in development with <a href="https://www.python.org/dev/peps/pep-0541/">PEP 541</a></p>
 
       <p>When <a href="https://www.python.org/dev/peps/pep-0541/">PEP 541</a> is accepted, the project will create a workflow, to be documented here.</p>
+
+      <h3 id="maintainers" class="common-question">Who maintains PyPI?</h3>
+
+      <p>Warehouse is an open source project developed under the umbrella of the Python Packaging Authority (PyPA) and supported by the Python Packaging Working Group (PackagingWG).</p>
+      <p>The <a href="https://www.pypa.io/en/latest/">PyPA</a> is an independent group of developers whose goal is to improve and maintain many of the core projects related to Python packaging.</p>
+      <p>The <a href="https://wiki.python.org/psf/PackagingWG">PackagingWG</a> is a working group of the Python Software Foundation (PSF) whose goal is to raise and disburse funds to support the ongoing improvement of Python packaging.</p>
+
+      <p>PyPI is powered by a variety of tools and services provided by our <a href="/sponsors">generous sponsors</a>.</p>
```

Comment:
Any internal links should use `request.route_path`, this should be:

```
<a href="{{ request.route_path('sponsors') }}">
```

Also I think this line would be better as it's own question, as our sponsors aren't really PyPI maintainers.